src/app/domain/upload/services/index.ts

```diff
@@ -0,0 +1,59 @@
+import { uploadFile } from '/app/domain/upload/services/upload'
+import {
+  UploadError,
+  UploadParams,
+  UploadResult
+} from '/app/domain/upload/models'
+
+export { uploadFile }
+
+import { SplitFilenameResult, models } from 'cozy-client'
+import { IOCozyFile } from 'cozy-client/types/types'
+
+export const uploadFileWithConflictStrategy = async ({
+  url,
+  token,
+  filename,
+  filepath,
+  mimetype,
+  notification
+}: UploadParams): Promise<UploadResult> => {
+  try {
+    return await uploadFile({
+      url,
+      token,
+      filename,
+      filepath,
+      mimetype,
+      notification
+    })
+  } catch (e) {
+    const error = e as UploadError
+
+    if (error.statusCode === 409) {
```

Comment:
nit: Would be great to have an explicitly named method like for other methods (`isQuotaExceededError`, `isFatalError` etc) so we easily understand that this catch is about name conflicts